src/pybind/mgr/dashboard/module.py

```diff
@@ -374,7 +377,7 @@ def fs_status(self, fs_id):
                 "id": fs_id,
                 "name": mdsmap['fs_name'],
                 "client_count": client_count,
-                "clients_url": "/clients/{0}/".format(fs_id),
```

Comment:
it would be nice to have a simple helper function to avoid doing the `+` everywhere